model/feature_flags.go

```diff
@@ -32,12 +32,16 @@ type FeatureFlags struct {
 	PluginPlaybooks  string `plugin_id:"playbooks"`
 	PluginApps       string `plugin_id:"com.mattermost.apps"`
 	PluginFocalboard string `plugin_id:"focalboard"`
+	PluginCalls      string `plugin_id:"com.mattermost.calls"`
```

Comment:
I /think/ most of these are now unused, but no objection to extending.

Reply:
Thanks, wasn't sure either but given there was a check that made use of those in `installFeatureFlagPlugins` I opted to include it although it shouldn't have any impact on the PR's scope.